code/modules/power/tesla/energy_ball.dm

```diff
@@ -174,8 +155,34 @@ var/list/blacklisted_tesla_types = typecacheof(list(/obj/machinery/atmospherics,
 	var/mob/living/closest_mob
 	var/obj/machinery/closest_machine
 	var/obj/structure/closest_structure
-
-	for(var/A in oview(source, zap_range+2))
+	var/static/list/blacklisted_types = typecacheof(list(
+		/obj/machinery/atmospherics,
+		/obj/machinery/power/emitter,
+		/obj/machinery/field_generator,
+		/mob/living/simple_animal,
```

Comment:
Why should simple animals not get zapped by the tesla ?

Reply:
Dunno, that was always there.

Reply:
late to the part here, but probably because Simple_animals didn't have an electrocute act when the tesla was first made.